jupyter_notebook_config.py

```diff
@@ -119,8 +120,14 @@ def script_post_save(model, os_path, contents_manager, **kwargs):
     if 'KBC_TOKEN' in os.environ:
         token = os.environ['KBC_TOKEN']
     else:
-        log.error("Could not find the keboola api token.")
-    response = saveFile(os.path.relpath(os_path), token)
-    log.info("Successfully saved to keboola")
+        log.error('Could not find the keboola api token.')
+        raise Exception('Could not find the keboola api token.')
```

Comment:
have a broken shift key? :trollface:

Reply:
IDGI

Reply:
how about "Could not find the Keboola Storage API token." ? :)

Reply:
aha,  I was counting spaces because I confused shift with tab and am dumb :-)

Reply:
lol